src/fairchem/core/_cli_hydra.py

```diff
@@ -79,24 +84,31 @@ class JobConfig:
     device_type: DeviceType = DeviceType.CUDA
     debug: bool = False
     scheduler: SchedulerConfig = field(default_factory=lambda: SchedulerConfig)
-    log_dir_name: str = "logs"
-    checkpoint_dir_name: str = "checkpoint"
-    config_file_name: str = "canonical_config.yaml"
     logger: Optional[dict] = None  # noqa: UP007 python 3.9 requires Optional still
     seed: int = 0
     deterministic: bool = False
+    runner_state: Optional[str] = None  # noqa: UP007 python 3.9 requires Optional still
```

Comment:
This is a path right? does it make sense to name it something like runner_state_path , just to make it obvious its the path and not the actual state?